code/script.js

```diff
@@ -1,19 +1,198 @@
+  }
+}
+//message with the food that has been chosen 
+alert(`Great choice, ${foodType}! Click 'OK' to select the specific type of ${foodType} you'd like.`)
 
 // Step 3 - Subtype choice
-// Your code goes here
+// Variables to store subtype choices
+let pizzaType
+let pastaType
+let saladType
+let subType
+
+//If choice is Pizza -> select type of pizza
+if (foodType === "Pizza") {
+  while (true) {
+    pizzaType = prompt(`Please select your type of Pizza 🍕: 
+    Write: 1 - for Margherita Pizza 
+    Write: 2 - for Pepperoni Pizza
+    Write: 3 - for Veggie Pizza`
+    )
+
+    pizzaType = Number(pizzaType)
+
+    if (pizzaType === 1) {
+      console.log("Margherita")
+      subType = "Margherita Pizza"
+      break
+    } else if (pizzaType === 2) {
+      console.log("Pepperoni Pizza")
+      subType = "Pepperoni"
+      break
+    } else if (pizzaType === 3) {
+      console.log("Veggie")
+      subType = "Veggie Pizza"
+      break
+    } else {
+      alert(`Oops! It looks like that's not an option. Please choose from 1, 2, or 3`)
+      console.log("Invalid selection. Please choose 1, 2, or 3.")
+    }
+  }
+}
+
+
+// If choice is pasta -> select type of pasta
+if (foodType === "Pasta") {
+  while (true) {
+    pastaType = prompt(`Please select your type of Pasta 🍝: 
+    Write: 1 - for Spaghetti Carbonara  
+    Write: 2 - for Spaghetti Bolognese
+    Write: 3 - for Pasta Primavera`
+    )
+
+    pastaType = Number(pastaType)
+
+    if (pastaType === 1) {
+      console.log("Carbonara")
+      subType = "Spaghetti Carbonara"
+      break
+    } else if (pastaType === 2) {
+      console.log("Bolognese")
+      subType = "Spaghetti Bolognese"
+      break
+    } else if (pastaType === 3) {
+      console.log("Primavera")
+      subType = "Pasta Primavera"
+      break
+    } else {
+      alert(`Oops! It looks like that's not an option. Please choose from 1, 2, or 3`)
+      console.log("Invalid selection. Please choose 1, 2, or 3.")
+    }
+  }
+}
+
+// If choice is salad -> select type of salad
+if (foodType === "Salad") {
+  while (true) {
+    saladType = prompt(`Please select your type of Salad 🥗: 
+    Write: 1 - for Ceacar Salad  
+    Write: 2 - for Shrimp Salad
+    Write: 3 - for Greek Salad`
+    )
+
+    saladType = Number(saladType)
+
+    if (saladType === 1) {
+      console.log("Ceacar Salad")
+      subType = "Ceacar Salad"
+      break
+    } else if (saladType === 2) {
+      console.log("Shrimp Salad")
+      subType = "Shrimp Salad"
+      break
+    } else if (saladType === 3) {
+      console.log("Greek Salad")
+      subType = "Greek Salad"
+      break
+    } else {
+      alert(`Oops! It looks like that's not an option. Please choose from 1, 2, or 3`)
+      console.log("Invalid selection. Please choose 1, 2, or 3.")
+    }
+  }
+}
+//message with the subtype that has been chosen 
+alert(`Yum! You've selected ${subType}! Excellent choice! Click 'OK' to start placing your order.`)
+
 
 // Step 4 - Age
```

Comment:
Smart to use the two options adult or child instead of age (integer). It made the response better, and the code cleaner.